cc-backend/openapi.yaml

```diff
@@ -0,0 +1,406 @@
+openapi: 3.0.3
+info:
+  version: 0.1.0
+  title: CC-Backend API-spec
+
+security:
+  - bearerAuth: []
+
+paths:
+  /v1/system/ping:
+    get:
+      operationId: ping
+      summary: Replies with pong
+      description: Replies ping with pong
+      tags:
+        - System
+      responses:
+        "200":
+          description: Pong
+          content:
+            text/plain:
+              schema:
+                type: string
+
+  /v1/user:
+    post:
+      operationId: afterLogin
+      summary: User has logged in
+      description: TODO
+      tags:
+        - User
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/UserInfo"
+        description: Payload
+        required: true
+      responses:
+        "200":
+          description: Added User info
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/UserInfo"
+        "400":
+          description: Invalid request was sent to the server
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        "403":
+          description: Attempted to make a call with unauthorized client
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        "404":
+          description: Not found
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        "500":
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+
+  /v1/items/{surveyId}:
+    get:
+      operationId: getItemsFromPK
+      summary: Get all items from PK
+      description: Gets all items from PK via CC-PK-service
```

Comment:
this request would also get all Items that the user has in the cc database via CC-db-service